e2e/tests/test_e2e.py

```diff
@@ -680,6 +680,42 @@ def test_x_cluster_deletion(self):
         self.assertEqual(0, k8s.count_pdbs_with_label(cluster_label))
         self.assertEqual(0, k8s.count_secrets_with_label(cluster_label))
 
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_decrease_max_connections(self):
+        '''
+            Test decreasing max_connections and restarting cluster through rest api
+        '''
+        k8s = self.k8s
+        cluster_label = 'application=spilo,cluster-name=acid-minimal-cluster'
+        labels = 'spilo-role=master,' + cluster_label
+        new_max_connections_value = "99"
+
+        # enable load balancer services
+        pg_patch_max_connections = {
+            "spec": {
+                "postgresql": {
+                    "parameters": {
+                        "max_connections": new_max_connections_value
+                     }
+                 }
+            }
+        }
+        k8s.api.custom_objects_api.patch_namespaced_custom_object(
+            "acid.zalan.do", "v1", "default", "postgresqls", "acid-minimal-cluster", pg_patch_max_connections)
+        # wait for service recreation
+        time.sleep(60)
+
+        pods = k8s.api.core_v1.list_namespaced_pod(
+            'default', label_selector=labels).items
+        self.assert_master_is_unique()
+        masterPod = pods[0]
+        get_max_connections_cmd = '''psql -At -U postgres -c "SELECT setting FROM pg_settings WHERE name = 'max_connections';"'''
+        result = k8s.exec_with_kubectl(masterPpod.metadata.name, get_max_connections_cmd)
```

Comment:
there is a typo here: `masterPpod` should be `masterPod`. Tests fail because of that
```
ERROR: test_decrease_max_connections (test_e2e.EndToEndTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/timeout_decorator/timeout_decorator.py", line 81, in new_function
    return function(*args, **kwargs)
  File "/test_e2e.py", line 713, in test_decrease_max_connections
    result = k8s.exec_with_kubectl(masterPpod.metadata.name, get_max_connections_cmd)
NameError: name 'masterPpod' is not defined
```